### PR TITLE
Added dev script for windows and tweaked the save tile menu layout

### PR DESCRIPTION
**Pull request by evanwporter**
Created a `npm run` dev command for windows, along with a corresponding powerscript. Since Windows can't run shell scripts. New commands are:
* `npm run dev:unix`
* `npm run dev:win`

Modified Aesthetics of Save Tile Menu
* Changed `Remove` icon to be consistent with `Open` icon on left hand side
* Moved Remove button to right hand side, this makes more sense to me, and I think it looks better

### CODE REVIEW

File: src/renderer/components/Icons.tsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-import { MdClose, MdCreateNewFolder, MdGrid4X4, MdMoreVert } from 'react-icons/md'
+import { IoGrid, IoCloseCircleOutline } from 'react-icons/io5'
+import { MdCreateNewFolder, MdGrid4X4, MdMoreVert } from 'react-icons/md'
+
+
 
 export const AddFolderIcon = MdCreateNewFolder
 export const MenuIcon = MdMoreVert
-export const RemoveIcon = MdClose
+export const RemoveIcon = IoCloseCircleOutline
```
Comment (andrewbenington) [inline src/renderer/components/Icons.tsx]: I like the original icon better, I think this looks weird inside a circular button

File: src/renderer/saves/RecentSaves.tsx [hunk 2]
```diff
@@ -126,6 +103,29 @@ export default function RecentSaves(props: SaveFileSelectorProps) {
...
+      width: 40,
+      renderCell: (params) => (
+        <button
+          style={{
+            padding: 0,
+            display: 'grid',
+            marginLeft: 'auto',
+            marginTop: 'auto',
+            marginBottom: 'auto',
+            // backgroundColor: '#990000',
```
Comment (andrewbenington) [inline src/renderer/saves/RecentSaves.tsx]: Can we leave this button red? That way it looks more like a "remove" button

**Review by andrewbenington — CHANGES_REQUESTED** [review]
Looks good aside from the two button things. Thanks for adding a windows version of the json script

**Comment by andrewbenington** [comment]
Also in future pull requests, can you request my review? You can do that by clicking the gear by "Reviewers" in the top right. It makes them easier for track which ones I need to look at

**Comment by evanwporter** [comment]
Yes I can do that

File: src/renderer/saves/RecentSaves.tsx (hunk 2 above)
Comment (evanwporter) [reply src/renderer/saves/RecentSaves.tsx]: Yeah for sure. I like the white background better but that can just be a local change

File: src/renderer/components/Icons.tsx (hunk 1 above)
Comment (evanwporter) [reply src/renderer/components/Icons.tsx]: Yeah for sure

**Review by andrewbenington — APPROVED** [review]
thanks for making the changes! merging now

**Pull request merged**